enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Federated Reporting
+published: ??
```

Comment:
published: true

Reply:
sorting i dunno, but if it's left out then it's just alphabetical sorting i think.

Here is the other sorting for the same section:
- install-get-started.markdown:sorting: 10
- user-interface.markdown:sorting: 20
- settings.markdown:sorting: 25
- hosts-health.markdown:sorting: 30
- alerts-notifications.markdown:sorting: 40
- alerts-custom-actions.markdown:sorting: 40
- reporting.markdown:sorting: 50
- monitoring.markdown:sorting: 60
- design-center.markdown:sorting: 70
- enterprise-api.markdown:sorting: 80
- best-practices.markdown:sorting: 100
- high-availability.markdown
- hub_administration.markdown